guides/data/git.md

```diff
@@ -1,8 +1,134 @@
+      }
+    },
+  }
+```
+
+Now if you start up your Card, you will be using the local git repository to store data! If you use the Edges to save data, and you run `git status` in your local repository, you will see its changes. You can also look at the files inside your data repository if you are curious to see what the Card's JSON.
+
+### Troubleshooting local repositories
+
+- *Any problem* - did you restart your server after making changes to the data sources?
+- *Can't start the app* - If you have trouble starting up your project after adding the git data source, double check that the path is correct and that you have at least one commit on the branch that git should target.
+- *My seed data is missing!* - When you define seed data for your project, it goes into the ephemeral data source. Therefore, you will not see that seed data if you have changed a Card's data source to git.
+- *Error, cannot locate local branch* - Make sure you have at least one commit on your target branch. If you decide to use the `branchPrefix` configuration, you will need to make a first commit on that branch as well. For example, if your `branchPrefix` is `cs-`, then make a commit on `cs-master`. If you are not using `branchPrefix`, make a commit to `master` and restart your server.
+
+## Remote data
+
+Now it is time to use a remote data source, such as a repository on [GitHub](https://github.com)! You will need a GitHub account with an [ssh key set up](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent). Although this example uses GitHub, you could use the git plugin with other remote hosts too.
+
+First, create a repository on GitHub and initialize it with a `README.md`:
+
+![screenshot showing the README option checked](/images/github-create-repo.png)
+
+Make sure to choose "private" if you don't want to expose your data with the world. Choose "public" if you want to share your work with the open source community.
+
+After you have created the repository, copy the `ssh` URL, such as `git@github.com:my-username/project-data.git`.
+
+Now in `cardhost/data-sources/default.js`, configure your project to use the remote repository. Make sure to replace `your-username` with your GitHub username or organization name:
+
+```javascript
+{
+    type: 'data-sources',
+    id: 'default',
+    attributes: {
+      'source-type': '@cardstack/git',
+      params: {
+        remote: {
+          url: 'git@github.com:your-username/project-template-data.git',
+          privateKey: process.env.GIT_PRIVATE_KEY,
+        }
+      }
+    }
+  }
+```
+
+*Never ever commit the file containing your private key.* If someone has your private key, they could delete anything on your GitHub, deploy malicious code, and other bad things.
+
+Always use an environmental variable to provide your key. You can do this from the command line:
+
+```bash
+GIT_PRIVATE_KEY="/path/to/your/private/key/id_rsa"
```

Comment:
is this supposed to be the path to the private key, or the private key itself? (I think it's the latter)